### PR TITLE
Fixed links to be in the Pulsar org

### PR DESCRIPTION
**Pull request by LandarXT**
This PR fixes the package links in Contributing.md to be to the Pulsar org's versions of those repos. I also renamed some of the other references.

### CODE REVIEW

**Review by confused-Techie — APPROVED** [review]
This looks great to me, although I recall someone on the server was putting together a very organized list of all our repos, and the purposes. Based off the pulsar-edit/.github repo, and if that project is still in development, this may be one of the best locations to link to it. 

So may be something to keep in mind after this is merged.

**Comment by LandarXT** [comment]
This was just a simple rebranding exercise, but I do intend to fully update this documentation soon, for example when the aforementioned list is finished, linking to that.

**Comment by confused-Techie** [comment]
If someone else from @pulsar-edit/documentation wants to approve this, I can go ahead and merge, but it looks good to me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ To get a sense for the packages that are bundled with Atom, you can go to `Setti
...
-* [apm](https://github.com/atom/apm) - the `apm` command line tool (Atom Package Manager). You should use this repository for any contributions related to the `apm` tool and for publishing packages.
-* [atom.io](https://github.com/atom/atom.io) - the repository for feedback on the [Atom.io website](https://atom.io) and the [Atom.io package API](https://github.com/atom/atom/blob/master/docs/apm-rest-api.md) used by [apm](https://github.com/atom/apm).
+* [pulsar-edit/pulsar](https://github.com/pulsar-edit/pulsar) - Atom Core! The core editor component is responsible for basic text editing (e.g. cursors, selections, scrolling), text indentation, wrapping, and folding, text rendering, editor rendering, file system operations (e.g. saving), and installation and auto-updating. You should also use this repository for feedback related to the [Atom API](https://atom.io/docs/api/latest) and for large, overarching design proposals.
```
Comment (mauricioszabo) [inline CONTRIBUTING.md:62]:
```suggestion
* [pulsar-edit/pulsar](https://github.com/pulsar-edit/pulsar) - Pulsar Core! The core editor component is responsible for basic text editing (e.g. cursors, selections, scrolling), text indentation, wrapping, and folding, text rendering, editor rendering, file system operations (e.g. saving), and installation and auto-updating. You should also use this repository for feedback related to the [API](https://atom.io/docs/api/latest) and for large, overarching design proposals.
```

**Comment by confused-Techie** [comment]
Since it looks like the concerns raised by @mauricioszabo have been resolved, I'll go ahead and merge into main.

**Comment by stech11845** [comment]
README has not been updated, updated in #11

**Pull request merged**